envoy/envoy.template.yaml

```diff
@@ -69,6 +69,32 @@ static_resources:
                               retry_on: '5xx,reset'
                               num_retries: 5
                             prefix_rewrite: '/'
+                        - match:
+                            prefix: '/chunk'
+                            headers:
+                              - name: ':method'
+                                exact_match: 'POST'
+                          route:
+                            cluster: ario_gateways
+                            timeout: 30s
+                            retry_policy: &post_retry_policy
```

Comment:
We don't need a retry policy here. The ar-io-node service handles that.

Reply:
removed